content/en/guide/v10/typescript.md

```diff
@@ -79,20 +78,7 @@ function MyComponent({ name, age }: MyComponentProps) {
 }
 ```
 
-You can set default props by setting a default value in the function signature.
-
-```tsx
-type GreetingProps = {
-  name?: string; // name is optional!
-}
-
-function Greeting({ name = "User" }: GreetingProps) {
-  // name is at least "User"
-  return <div>Hello {name}!</div>
-}
-```
```

Comment:
This is explanation for Default parameters.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters
Therefore, This is not appropriate in TypeScript page of Preact document.

Reply:
I think the section was great and we should keep it. People switching from prop types over to TS will naturally ask these questions as most linters treat prop-types and default props as the same. We should cater to those users too :+1: